common/tools/eslint-plugin-azure-sdk/tests/rules/ts-package-json-engine-is-present.ts

```diff
@@ -249,108 +249,105 @@ const ruleTester = new RuleTester({
   parser: require.resolve("@typescript-eslint/parser"),
   parserOptions: {
     createDefaultProgram: true,
-    project: "./tsconfig.json",
-  },
+    project: "./tsconfig.json"
```

Comment:
Changes in this file is likely because different versions of prettier.  Since it is unrelated, please undo.

Reply:
The changes are reverted in 94e4405